<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PBoreSinePole</class>
 <widget class="QWidget" name="PBoreSinePole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="img_bore">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../Resources/pyleecan.qrc">:/images/images/MachineSetup/LamParam/BoreSinePole.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>268</width>
        <height>450</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="in_N">
           <property name="text">
            <string>N</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="si_N">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="unit_N">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="in_W0">
           <property name="text">
            <string>W0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="FloatEdit" name="lf_W0"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="unit_W0">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="in_k">
           <property name="text">
            <string>k</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="FloatEdit" name="lf_k"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="unit_k">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="in_delta_d">
           <property name="text">
            <string>delta_d</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="FloatEdit" name="lf_delta_d"/>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="unit_delta_d">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="in_delta_q">
           <property name="text">
            <string>delta_q</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="FloatEdit" name="lf_delta_q"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="unit_delta_q">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="in_alpha">
           <property name="text">
            <string>alpha</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="FloatEdit" name="lf_alpha"/>
         </item>
         <item row="5" column="2">
          <widget class="QComboBox" name="c_alpha_unit">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>[rad]</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>[deg]</string>
            </property>
           </item>
          </widget>
         </item>   
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="WBoreOut" name="w_out" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatEdit</class>
   <extends>QLineEdit</extends>
   <header>......GUI/Tools/FloatEdit.h</header>
  </customwidget>
  <customwidget>
   <class>WBoreOut</class>
   <extends>QWidget</extends>
   <header>......GUI.Dialog.DMachineSetup.DBore.WBoreOut.WBoreOut.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lf_W0</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
